### DISCUSSION TITLE
Not accepting an IP-Adress on STM32 H753ZI

### DISCUSSION THREAD

**Discussion by AlphaApfel** [cesanta/mongoose#2773] [Ask questions, get answers] [answered]
I implemented the software on an STM32 H753ZI Nucleo. When I flash the programm to the board it is requesting an IP. The problem I have is that it wont accept it. I´m using haneWIN as a DHCP-Server. I have configured the right pins for ETH. 

Has someone an idea how to fix the problem?

**Comment by scaprile** [comment, marked as answer]
This is the list of our STM32 boards with tutorials https://mongoose.ws/documentation/#stm32
This is the list of our examples for STM32 https://github.com/cesanta/mongoose/tree/master/examples/stm32
There is no Nucleo H753 there,
That means that we haven't tried your board, so you're on your own.
The closest I can offer is H755 in our Wizard https://mongoose.ws/wizard/#/ it is beta software.

If you say the board is requesting an IP and don't accepting it, I imagine that you've seen the DHCP dialog. Then you say you "configured the right pins for ETH"... Of course you did, otherwise there won't be no DHCP dialog. So... did you ? Can you positively say the link is up ? Did you sniff your network ? Did you actually see the DHCP dialog ?
Then you say you "implemented the software". What software ? Built-in stack ? lwIP ? FreeRTOS+TCP ? RTX ? RTX5?
If you are using our built-in stack, what controller does that board have ? How can you know if our driver supports it ? Are you sure your pin selection is correct ? Did you use the proper AF ?
